clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/config/AuditLoggingConfiguration.java

```diff
@@ -31,31 +32,28 @@
  */
 @Data
 @NoArgsConstructor
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
```

Comment:
Nit: we can apply naming strategy globally [here](https://github.com/abcxyz/lumberjack/blob/ae3fc6bb1b9eea6a3344d4be72eac617a75422c9/clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/modules/AuditLoggingConfigurationModule.java#L38) instead of per class.
```
mapper.setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE);
```

Reply:
Done.